Assignment1/pages/game.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <v-card class="mx-auto my-8" elevation="16" max-width="430">
+    <v-card-item>
+      <v-card-title> "Marco!" </v-card-title>
+      <v-card-subtitle> What will you reply? </v-card-subtitle>
+    </v-card-item>
+    <v-card-text>
+      <v-text-field
+        v-model="myGuess"
+        label="You say:"
+        outlined
+        clearable
+      >
+      </v-text-field>
```

Comment:
```suggestion
      <v-text-field
        v-model="myGuess"
        label="You say:"
        outlined
        clearable
      />
```
When an element has no child content, you can self-close it.